packages/unplugin-typia/src/core/typia.ts

```diff
@@ -116,24 +117,18 @@ async function getProgramAndSource(
 
 	host.resolveModuleNameLiterals = (moduleLiterals, containingFile) => {
 		return moduleLiterals.map((lit) => {
+			let module = ts.resolveModuleName(lit.text, containingFile, compilerOptions, host);
 			let alias;
 			// eslint-disable-next-line no-cond-assign
-			if (aliases && !lit.text.startsWith('./') && (alias = findMatchingAlias(lit.text, aliases))) {
-				let extension;
-				if (lit.text.endsWith('.d.ts')) {
-					extension = '.d.ts';
-				}
-				else {
-					extension = path.extname(lit.text);
-				}
-				return {
-					resolvedModule: {
-						resolvedFileName: path.resolve(lit.text.replace(alias.find, alias.replacement)),
-						extension,
-					},
-				};
+			if (!module.resolvedModule && aliases && (alias = findMatchingAlias(lit.text, aliases))) {
+				module = ts.resolveModuleName(
```

Comment:
Use ts native module resolution, this doesn't need extension anymore

Reply:
So, after this change, which aliases are resolved, defined in vite.config.ts or tsconfig.json

Reply:
It's always aliases from vite.config.ts but now it first tries to resolve with native ts module resolution. If it fails and import is present in path aliases it uses ts.resolveModuleName again with updated path. Resolving from tsconfig should not be made here because tsc and vite does not do it so having it here would be unexpected to users. It could be added as another separate feature behind a plugin config flag if we want to support it.

Reply:
Ok, got it thanks!